setup.py

```diff
@@ -1,73 +1,102 @@
 #!/usr/bin/env python3
+import io
 import os.path
+import re
 import stat
 import sys
+import tarfile
+import tempfile
+import zipfile
 from distutils.command.build import build as orig_build
 from distutils.core import Command
-from hashlib import sha512
 from http import HTTPStatus
 from urllib.request import urlopen
 
 from setuptools import setup
 from setuptools.command.install import install as orig_install
 
-SHELLCHECK_VERSION = '0.7.0'
+SHELLCHECK_VERSION = '0.7.1'
 PY_VERSION = '1'
 
 
 def get_arch() -> str:
     if sys.platform.startswith('linux'):
-        # TODO(rhee): detect "linux-aarch64" and "linux-armv6hf"
-        return 'linux-x86_64'
+        # TODO(rhee): detect "linux.aarch64" and "linux.armv6hf"
```

Comment:
I'm not sure changing this comment is correct, what are the architecture strings?

Reply:
I think it is, previously the separator in downloadables was a dash, now it's a period. But YMMV.